content/en/docs/demo/docker-deployment.md

```diff
@@ -78,24 +75,26 @@ with an editor.
       endpoint: <your-endpoint-url>
   ```
 
-- Then add a new pipeline with your new exporter:
+- Then override the `exporters` for telemetry pipelines that you want to use for
+  your backend.
 
   ```yaml
   service:
     pipelines:
       traces:
-        receivers: [otlp]
-        processors: [batch]
-        exporters: [otlphttp/example]
+        exporters: [spanmetrics, otlphttp/example]
   ```
 
+> **Note:** When merging YAML values with the Collector, objects are merged and
+> arrays are replaced. The `spanmetrics` exporter must be included in the array
+> of exporters for the `traces` pipeline if overridden. Not including this
+> exporter will result in an error.
+
 Vendor backends might require you to add additional parameters for
 authentication, please check their documentation. Some backends require
 different exporters, you may find them and their documentation available at
 [opentelemetry-collector-contrib/exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter).
 
 After updating the `otelcol-config-extras.yml`, start the demo by running
-`docker compose up`[^1]. After a while, you should see the traces flowing into
-your backend as well.
-
-[^1]: {{% _param notes.docker-compose-v2 %}}
```

Comment:
please keep that foot note

Reply:
Why are we keeping the footnote here? It references `docker-compose`, which we are not using, and have clear as a pre-requisite at the top to use docker compose v2.0+

Reply:
We have made the experience that people do not read the pre-requisites, run that code on a machine with old docker compose, get stuck and raise an issue. 

If you think that enough time has passed to remove that footnote, then we should remove it at all the places and solve this via a separate issue/PR, until then let's please keep it whenever we use `docker compose`